### PR TITLE
Added more tests for storage helpers

### PR DESCRIPTION
**Pull request by mrsaicharan1**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
- added test for create_url method

 #### Changes proposed in this pull request:

- Added more tests for storage helpers

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Very small change. Add more tests

**Comment by mrsaicharan1** [comment]
@iamareebjamal Yes, will be writing more tests for it

**Comment by mrsaicharan1** [comment]
@iamareebjamal Please review

File: tests/all/integration/api/helpers/test_storage.py [hunk 1]
```diff
@@ -32,3 +33,17 @@ def test_https_url(self):
         self.assertEqual(
             expected_file_url, create_url(request_url, '/some/path/image.png')
             )
+
+    def test_create_url(self):
```
Comment (iamareebjamal) [inline tests/all/integration/api/helpers/test_storage.py]: Move to unit test

**Comment by mrsaicharan1** [comment]
@iamareebjamal Moved the test to the right place

File: tests/all/unit/api/helpers/test_storage.py [hunk 2]
```diff
@@ -32,3 +33,17 @@ def test_https_url(self):
...
+        """Method to test a generated url of uploaded file"""
+        request_url = "https://localhost:4200"
+        expected_file_url = 'https://localhost:4200/some/path/image.png'
+        self.assertEqual(
+            expected_file_url, create_url(request_url, '/some/path/image.png')
+            )
+
+    def test_generate_hash(self):
+        """Method to test generation of hash with key"""
+        with app.test_request_context():
```
Comment (iamareebjamal) [inline tests/all/unit/api/helpers/test_storage.py]: Remove this, this is a unit test

Comment (mrsaicharan1) [reply tests/all/unit/api/helpers/test_storage.py]: Isn't this test is a part of unit test?

Comment (iamareebjamal) [reply tests/all/unit/api/helpers/test_storage.py]: This bootstraps the flask app. Unit Test by definition is pure function not depending on framework/database/network

Comment (mrsaicharan1) [reply tests/all/unit/api/helpers/test_storage.py]: @iamareebjamal Got it.

**Comment by mrsaicharan1** [comment]
@iamareebjamal @CosmicCoder96 Please review :)

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 Please review :)

**Pull request merged**